<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectionView</class>
 <widget class="QWidget" name="ProjectionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QSlider" name="verticalSlider">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>view:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="_L_slice">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="WindowingWidget" name="_W_windowing" native="true"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Zoom factor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="_SB_zoom">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="suffix">
          <string> x</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>39</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>294</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="_L_image">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WindowingWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/windowingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ProjectionView</receiver>
   <slot>updateImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_SB_zoom</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ProjectionView</receiver>
   <slot>updateImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateImage()</slot>
 </slots>
</ui>
